server/mainapp/models.py

```diff
@@ -123,3 +123,183 @@ def fetch_data(self) -> response.JsonResponse:
             return docs
 
         raise DataFetchingError("There Are No Posts In The Database At This Moment")
+
+
+class Likes:
+    def __init__(self) -> None:
+        """
+        Connect to MongoDB
+        """
+        client = pymongo.MongoClient(DATABASE["mongo_uri"])
+        self.db = client[DATABASE["db"]]["Likes"]
+
+    def insert_data(
```

Comment:
It might be better to rename this function to "add_like" or "increment_like" - It is more apt for the functionality it is providing. Also have you kept a track of the number of likes (e.g a counter) so that when displaying it on the client side, a response with the number of likes per post can be sent